Makefile

```diff
@@ -8,6 +8,8 @@
 # You may select, at your option, one of the above-listed licenses.
 # ################################################################
 
+Q = $(if $(filter 1,$(V) $(VERBOSE)),,@)
```

Comment:
Maybe we should have a comment about what `Q` is for or a more descriptive variable name?

Reply:
This is an `automake` convention (part of [GNU autotools](https://www.gnu.org/software/automake/)).
`cmake` offers essentially the same thing, though it prefers `VERBOSE=1` (rather than `V=1`).

To be fair, this is more a debug capability, not something normal users are expected to invoke.

We can certainly comment, we just have to avoid going too far and become an `autotool` tutorial.

Reply:
Ah I see. Thanks for the explanation. Was't familar with this convention